### PR TITLE
Decode morse code

### PR DESCRIPTION
**Pull request by preciousuwhubetine**
In this branch, I and @Tafloninno did the following:
- Wrote a function to decode a morse coded character.
- Wrote a function to decode a morse coded word.
- Wrote a function to decode a morse coded string.

### CODE REVIEW

File: morse_code.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    '.' => 'E', '..-.' => 'F', '--.' => 'G', '....' => 'H',
+    '..' => 'I', '.---' => 'J', '-.-' => 'K', '.-..' => 'L',
+    '--' => 'M', '-.' => 'N', '---' => 'O', '.--.' => 'P',
+    '--.-' => 'Q', '.-.' => 'R', '...' => 'S', '-' => 'T',
+    '..-' => 'U', '...-' => 'V', '.--' => 'W', '-..-' => 'X',
+    '-.--' => 'Y', '--..' => 'Z'
+  }
+  morse_dict[morse_code].upcase
+end
+
```
Comment (amejid) [inline morse_code.rb:13]: - [ ] You did a great job working on `decode_char` and `decode` methods. :+1: It seems the implementation for `decode_word` is missing. Kindly create the function to achieve the project requirements. 🤝️

**Requirements**
image

**Review by amejid — CHANGES_REQUESTED** [review]
Hi team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- Linter check passing :heavy_check_mark: 
- Correct workflow :heavy_check_mark: 
- Decoding a character and a message working fine :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: morse_code.rb (hunk 1 above)
Comment (Whoistolu) [reply morse_code.rb:13]: - Well-resolved team 👍🏽

**Review by Whoistolu — APPROVED** [review]
### Hi Team,

Great job implementing the recommended changes by the previous reviewer.
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# STATUS: APPROVED 🟢
**Congratulations! 🎉** 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**